src/unity.yml

```diff
@@ -154,12 +158,16 @@ components:
       $ref: './unity/schemas/Me.yml'
     OrgUsers:
       $ref: './unity/schemas/OrgUsers.yml'
-    Organization:
-      $ref: './unity/schemas/Organization.yml'
-    Organizations:
-      $ref: './unity/schemas/Organizations.yml'
+    OperatorOrganization:
+      $ref: './unity/schemas/OperatorOrganization.yml'
+    OperatorOrganizations:
+      $ref: './unity/schemas/OperatorOrganizations.yml'
     RelatedAccount:
       $ref: './unity/schemas/RelatedAccount.yml'
+    Organization:
+      $ref: './unity/schemas/Organization.yml'
+    OrganizationSummaries:
```

Comment:
This is just a question for my benefit, why do we include `OrganizationSummaries` (plural) here but not `OrganizationSummary` (singular)? Do we only need to include the top-level schemas that are referenced by the paths?